docs/how-to/use-flask-extension.rst

```diff
@@ -0,0 +1,49 @@
+Using the flask extension
+-------------------------
+
+The Flask extension is compatible with ``bare``, ``ubuntu:20.04``, and
+``ubuntu:22.04``. To employ it, include ``extensions: [flask]`` in your
+``rockcraft.yaml`` file.
+
+Example:
+
+.. code-block:: yaml
+
+    name: example
+    summary: Example.
+    description: Example.
+    version: "0.1"
+    base: bare
+    license: Apache-2.0
+
+    extensions:
+      - flask
+
+Managing project files with the flask extension
+-----------------------------------------------
+
+By default, all files within the Flask project directory are copied, excluding
+certain common files and directories, such as ``node_modules``. However,
+this behaviour can be tailored to either specifically include or exclude files
+from the Flask project directory in the rock image.
+
+To include only select files from the project directory in the rock image,
+append the following part to ``rockcraft.yaml``:
```

Comment:
I think the example is good but we should explain the `prime` bit just _a bit_ to give some necessary context. Something like:

> You can include and exclude files from the project directory in the ROCK image by using the standard `prime` declaration on the specially-named `flask/install-app` part. For example, to include only select files:

(and then)

> or to exclude specific files: